scripts/assert-release-consistency.mjs

```
@@ -0,0 +1,84 @@
+  if (changelog !== undefined && !changelog.includes(`## [${version}]`)) {
+    problems.push(
+      `CHANGELOG.md has no "## [${version}]" entry for package.json version "${version}"`,
+    );
+  }
+
+  return problems;
+}
+
+function readJson(relativePath) {
+  return JSON.parse(
+    readFileSync(new URL(relativePath, import.meta.url), "utf8"),
```

ESLint

[error] 'URL' is not defined.

(no-undef)